parsers/datalog/idlv/idlv_parser_visitor_implementation.py

```diff
@@ -0,0 +1,51 @@
+
+from antlr4 import PredictionMode
+from antlr4.CommonTokenStream import CommonTokenStream
+from antlr4.error.ErrorListener import ConsoleErrorListener
+from antlr4.error.Errors import RecognitionException
+from antlr4.error.ErrorStrategy import BailErrorStrategy, DefaultErrorStrategy
+from antlr4.InputStream import InputStream
+
+from languages.datalog.minimal_model import MinimalModel
+from .IDLVLexer import IDLVLexer
+from .IDLVParser import IDLVParser
+from .IDLVParserVisitor import IDLVParserVisitor
+
+
+class IDLVParserVisitorImplementation(IDLVParserVisitor):
+    def __init__(self, models):
+        self._models = models
+        self._model_currently_being_visited = None
+
+    def visitMinimal_model(self, ctx):
+        self._model_currently_being_visited = MinimalModel(set())
+        self._models.add_minimal_model(self._model_currently_being_visited)
+        return self.visitChildren(ctx)
+
+    def visitPredicate_atom(self, ctx):
+        self._model_currently_being_visited.get_atoms_as_stringlist().add(ctx.getText())
+
+    @staticmethod
+    def parse(answerSets, dlv2Output, two_stageParsing):
```

Comment:
Typo on this line, should be minimalModels instead of answerSets and idlvOutput instead of dlv2Output

Reply:
Yes, you should fix it